astrodb_utils/sources.py

```diff
@@ -224,8 +228,8 @@ def ingest_names(
 def ingest_source(
     db,
     source,
-    *,
     reference: str = None,
+    *,
     ra: float = None,
```

Comment:
Isn't the point of `*` in the args to enforce explicit use of keyword arguments? So shouldn't it remain above reference?